<!DOCTYPE html>

<html>

<head>

<meta charset="utf-8" />
<meta name="generator" content="pandoc" />
<meta http-equiv="X-UA-Compatible" content="IE=EDGE" />




<title>Supplementary Table 2 - Phylogeographic models</title>

<script src="site_libs/header-attrs-2.11/header-attrs.js"></script>
<script src="site_libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/paper.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<style>h1 {font-size: 34px;}
       h1.title {font-size: 38px;}
       h2 {font-size: 30px;}
       h3 {font-size: 24px;}
       h4 {font-size: 18px;}
       h5 {font-size: 16px;}
       h6 {font-size: 12px;}
       code {color: inherit; background-color: rgba(0, 0, 0, 0.04);}
       pre:not([class]) { background-color: white }</style>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/highlightjs-9.12.0/textmate.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>
<script src="site_libs/kePrint-0.0.1/kePrint.js"></script>
<link href="site_libs/lightable-0.0.1/lightable.css" rel="stylesheet" />

<style type="text/css">
  code{white-space: pre-wrap;}
  span.smallcaps{font-variant: small-caps;}
  span.underline{text-decoration: underline;}
  div.column{display: inline-block; vertical-align: top; width: 50%;}
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
  ul.task-list{list-style: none;}
    </style>

<style type="text/css">code{white-space: pre;}</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>








<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
img {
  max-width:100%;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
pre code {
  padding: 0;
}
</style>


<style type="text/css">
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #adb5bd;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script type="text/javascript">
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark it active
  menuAnchor.tab('show');

  // if it's got a parent navbar menu mark it active as well
  menuAnchor.closest('li.dropdown').addClass('active');

  // Navbar adjustments
  var navHeight = $(".navbar").first().height() + 15;
  var style = document.createElement('style');
  var pt = "padding-top: " + navHeight + "px; ";
  var mt = "margin-top: -" + navHeight + "px; ";
  var css = "";
  // offset scroll position for anchor links (for fixed navbar)
  for (var i = 1; i <= 6; i++) {
    css += ".section h" + i + "{ " + pt + mt + "}\n";
  }
  style.innerHTML = "body {" + pt + "padding-bottom: 40px; }\n" + css;
  document.head.appendChild(style);
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "&#xe258;";
  border: none;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
  background-color: transparent;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<!-- code folding -->




</head>

<body>


<div class="container-fluid main-container">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">Scoping review</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li class="dropdown">
  <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-expanded="false">
    Supplementary Tables
     
    <span class="caret"></span>
  </a>
  <ul class="dropdown-menu" role="menu">
    <li>
      <a href="st1.html">ST1 - Description of studies</a>
    </li>
    <li>
      <a href="st2.html">ST2 - Phylodynamic models</a>
    </li>
    <li>
      <a href="st3.html">ST3 - Mathematical models</a>
    </li>
    <li>
      <a href="st4.html">ST4 - Interdisciplinary studies</a>
    </li>
    <li>
      <a href="st5.html">ST5 - Estimated parameters in phylodynamics</a>
    </li>
    <li>
      <a href="st6.html">ST6 - Estimated parameters in mathematical models</a>
    </li>
  </ul>
</li>
<li>
  <a href="supp_text.html">Supplementary Text</a>
</li>
<li>
  <a href="prisma.html">PRISMA statement</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div id="header">



<h1 class="title toc-ignore">Supplementary Table 2 - Phylogeographic models</h1>

</div>


<p>Description of the phylogeographic models with an emphasis on data source and quality.</p>
<p>Supplementary Table 2 is also available in Excel format <a href="https://github.com/mlayan/RabiesScopingReview/blob/gh-pages/supplementary/S2_Table.xlsx">here</a>.</p>
<div style="border: 1px solid #ddd; padding: 0px; overflow-y: scroll; height:800px; overflow-x: scroll; width:100%; ">
<table class="table" style="margin-left: auto; margin-right: auto;border-bottom: 0;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;color: black !important;vertical-align: middle !important;position: sticky; top:0; background-color: #FFFFFF;">
Phylodynamics category
</th>
<th style="text-align:left;font-weight: bold;color: black !important;vertical-align: middle !important;position: sticky; top:0; background-color: #FFFFFF;">
Model description
</th>
<th style="text-align:left;font-weight: bold;color: black !important;vertical-align: middle !important;position: sticky; top:0; background-color: #FFFFFF;">
Software
</th>
<th style="text-align:left;font-weight: bold;color: black !important;vertical-align: middle !important;position: sticky; top:0; background-color: #FFFFFF;">
Genetic sequences
</th>
<th style="text-align:left;font-weight: bold;color: black !important;vertical-align: middle !important;position: sticky; top:0; background-color: #FFFFFF;">
Sequence length (pb)
</th>
<th style="text-align:left;font-weight: bold;color: black !important;vertical-align: middle !important;position: sticky; top:0; background-color: #FFFFFF;">
Data source
</th>
<th style="text-align:left;font-weight: bold;color: black !important;vertical-align: middle !important;position: sticky; top:0; background-color: #FFFFFF;">
Parameters inferred
</th>
<th style="text-align:left;font-weight: bold;color: black !important;vertical-align: middle !important;position: sticky; top:0; background-color: #FFFFFF;">
Reference
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;color: black !important;">
Parsimony discrete phylogeography
</td>
<td style="text-align:left;color: black !important;">
Maximum parsimony
</td>
<td style="text-align:left;color: black !important;">
PAUP*
</td>
<td style="text-align:left;color: black !important;">
N (n = 130)
</td>
<td style="text-align:left;color: black !important;">
1335
</td>
<td style="text-align:left;color: black !important;">
Not informed
</td>
<td style="text-align:left;color: black !important;">
Nucleotide substitution rate, Migration rates, tMRCA
</td>
<td style="text-align:left;color: black !important;">
<a href="https://doi.org/10.1099/vir.0.2008/003913-0">Bourhy et al., 2008</a>
</td>
</tr>
<tr>
<td style="text-align:left;color: black !important;">
Bayesian discrete phylogeography
</td>
<td style="text-align:left;color: black !important;">
Symmetric DTA + BSSVS
</td>
<td style="text-align:left;color: black !important;">
BEAST 1
</td>
<td style="text-align:left;color: black !important;">
N (n = 101)
</td>
<td style="text-align:left;color: black !important;">
1335
</td>
<td style="text-align:left;color: black !important;">
Bourhy et al., 2008
</td>
<td style="text-align:left;color: black !important;">
Migration rates
</td>
<td style="text-align:left;color: black !important;">
<a href="https://doi.org/10.1371/journal.pcbi.1000520">Lemey et al., 2009</a>
</td>
</tr>
<tr>
<td style="text-align:left;color: black !important;">
Parsimony discrete phylogeography
</td>
<td style="text-align:left;color: black !important;">
Maximum parsimony
</td>
<td style="text-align:left;color: black !important;">
PAUP*
</td>
<td style="text-align:left;color: black !important;">
N (n = 231)<br>G (n = 34)
</td>
<td style="text-align:left;color: black !important;">
1335<br>1572
</td>
<td style="text-align:left;color: black !important;">
Passive
</td>
<td style="text-align:left;color: black !important;">
Nucleotide substitution rate, Migration rates, tMRCA
</td>
<td style="text-align:left;color: black !important;">
<a href="https://doi.org/10.1099/vir.0.007765-0">Talbi et al., 2009</a>
</td>
</tr>
<tr>
<td style="text-align:left;color: black !important;">
Parsimony discrete phylogeography
</td>
<td style="text-align:left;color: black !important;">
DELTRAN and ACCTRAN optimization<br>Monte Carlo simulations
</td>
<td style="text-align:left;color: black !important;">
MigraPhyla
</td>
<td style="text-align:left;color: black !important;">
N (n = 200)
</td>
<td style="text-align:left;color: black !important;">
1353
</td>
<td style="text-align:left;color: black !important;">
Passive<br>GenBank
</td>
<td style="text-align:left;color: black !important;">
Nucleotide substitution rate, Migration rates, tMRCA
</td>
<td style="text-align:left;color: black !important;">
<a href="https://doi.org/10.1016/j.virol.2010.12.011">Meng et al., 2011</a>
</td>
</tr>
<tr>
<td style="text-align:left;color: black !important;">
Bayesian discrete phylogeography
</td>
<td style="text-align:left;color: black !important;">
DTA + BSSVS<sup>a</sup>
</td>
<td style="text-align:left;color: black !important;">
BEAST 1
</td>
<td style="text-align:left;color: black !important;">
N (n = 303)
</td>
<td style="text-align:left;color: black !important;">
405
</td>
<td style="text-align:left;color: black !important;">
Passive<br>GenBank
</td>
<td style="text-align:left;color: black !important;">
Migration rates, Root state, tMRCA
</td>
<td style="text-align:left;color: black !important;">
<a href="https://doi.org/10.1371/journal.pntd.0001001">Hayman et al., 2011</a>
</td>
</tr>
<tr>
<td style="text-align:left;color: black !important;">
Bayesian discrete phylogeography
</td>
<td style="text-align:left;color: black !important;">
Symmetric DTA + BSSVS
</td>
<td style="text-align:left;color: black !important;">
BEAST 1
</td>
<td style="text-align:left;color: black !important;">
N (n = 71)
</td>
<td style="text-align:left;color: black !important;">
1332
</td>
<td style="text-align:left;color: black !important;">
GenBank
</td>
<td style="text-align:left;color: black !important;">
Nucleotide substitution rate, Migration rates, tMRCA
</td>
<td style="text-align:left;color: black !important;">
<a href="https://doi.org/10.1007/s00705-011-0942-y">Carnieli et al., 2011</a>
</td>
</tr>
<tr>
<td style="text-align:left;color: black !important;">
Parsimony discrete phylogeography
</td>
<td style="text-align:left;color: black !important;">
DELTRAN optimization<br>Monte Carlo simulations
</td>
<td style="text-align:left;color: black !important;">
PAUP*<br>MigraPhyla
</td>
<td style="text-align:left;color: black !important;">
N (n = 210)
</td>
<td style="text-align:left;color: black !important;">
720
</td>
<td style="text-align:left;color: black !important;">
Active<br>Passive
</td>
<td style="text-align:left;color: black !important;">
Nucleotide substitution rate, Migration rates, tMRCA
</td>
<td style="text-align:left;color: black !important;">
<a href="https://doi.org/10.1371/journal.pntd.0001640">Yu et al., 2012</a>
</td>
</tr>
<tr>
<td style="text-align:left;color: black !important;">
Bayesian discrete phylogeography
</td>
<td style="text-align:left;color: black !important;">
Asymmetric DTA + BSSVS
</td>
<td style="text-align:left;color: black !important;">
BEAST 1
</td>
<td style="text-align:left;color: black !important;">
intergenic G-L (n = 635)
</td>
<td style="text-align:left;color: black !important;">
590
</td>
<td style="text-align:left;color: black !important;">
GenBank
</td>
<td style="text-align:left;color: black !important;">
Migration rates, Root state, tMRCA
</td>
<td style="text-align:left;color: black !important;">
<a href="https://doi.org/10.1007/s11262-013-0974-3">Mollentze et al., 2013</a>
</td>
</tr>
<tr>
<td style="text-align:left;color: black !important;">
Bayesian discrete phylogeography
</td>
<td style="text-align:left;color: black !important;">
Symmetric DTA + BSSVS
</td>
<td style="text-align:left;color: black !important;">
BEAST 1
</td>
<td style="text-align:left;color: black !important;">
N (n = 232)
</td>
<td style="text-align:left;color: black !important;">
1353
</td>
<td style="text-align:left;color: black !important;">
Active<br>Passive
</td>
<td style="text-align:left;color: black !important;">
Nucleotide substitution rate, Migration rates, tMRCA
</td>
<td style="text-align:left;color: black !important;">
<a href="https://doi.org/10.1371/journal.pntd.0002039">Guo et al., 2013</a>
</td>
</tr>
<tr>
<td style="text-align:left;color: black !important;">
Bayesian continuous phylogeography
</td>
<td style="text-align:left;color: black !important;">
Relaxed random walk (lognormal prior)
</td>
<td style="text-align:left;color: black !important;">
BEAST 1
</td>
<td style="text-align:left;color: black !important;">
N (n = 53)
</td>
<td style="text-align:left;color: black !important;">
1388
</td>
<td style="text-align:left;color: black !important;">
Passive<br>GenBank
</td>
<td style="text-align:left;color: black !important;">
Nucleotide substitution rate, Lineage velocity, tMRCA
</td>
<td style="text-align:left;color: black !important;">
<a href="https://doi.org/10.1007/s00705-013-1755-y">Carnieli et al., 2013</a>
</td>
</tr>
<tr>
<td style="text-align:left;color: black !important;">
Bayesian discrete phylogeography
</td>
<td style="text-align:left;color: black !important;">
Symmetric DTA + BSSVS
</td>
<td style="text-align:left;color: black !important;">
BEAST 1
</td>
<td style="text-align:left;color: black !important;">
G (n = 263)
</td>
<td style="text-align:left;color: black !important;">
1572
</td>
<td style="text-align:left;color: black !important;">
Passive
</td>
<td style="text-align:left;color: black !important;">
Nucleotide substitution rate, Markov jumps, Migration rates, tMRCA
</td>
<td style="text-align:left;color: black !important;">
<a href="https://doi.org/10.1016/j.meegid.2014.01.026">Tohma et al., 2014</a>
</td>
</tr>
<tr>
<td style="text-align:left;color: black !important;">
Bayesian discrete phylogeography
</td>
<td style="text-align:left;color: black !important;">
Symmetric DTA + BSSVS
</td>
<td style="text-align:left;color: black !important;">
BEAST 1
</td>
<td style="text-align:left;color: black !important;">
N (n = 139)
</td>
<td style="text-align:left;color: black !important;">
400
</td>
<td style="text-align:left;color: black !important;">
Passive<br>Systematic
</td>
<td style="text-align:left;color: black !important;">
Markov jumps, Migration rates, Root state, tMRCA
</td>
<td style="text-align:left;color: black !important;">
<a href="https://doi.org/10.1371/journal.pntd.0003569">Horton et al., 2015</a>
</td>
</tr>
<tr>
<td style="text-align:left;color: black !important;">
Bayesian discrete phylogeography
</td>
<td style="text-align:left;color: black !important;">
Asymmetric DTA + BSSVS
</td>
<td style="text-align:left;color: black !important;">
BEAST 1
</td>
<td style="text-align:left;color: black !important;">
partial N gene (n = 430)<br>N (n = 100)<br>whole genome (n = 58)
</td>
<td style="text-align:left;color: black !important;">
405<br>1350<br>-
</td>
<td style="text-align:left;color: black !important;">
Active<br>Passive
</td>
<td style="text-align:left;color: black !important;">
Nucleotide substitution rate, Markov jumps, Migration rates, Population dynamics
</td>
<td style="text-align:left;color: black !important;">
<a href="https://doi.org/10.1093/ve/vev011">Brunker et al., 2015</a>
</td>
</tr>
<tr>
<td style="text-align:left;color: black !important;">
Bayesian discrete phylogeography
</td>
<td style="text-align:left;color: black !important;">
Symmetric DTA + BSSVS
</td>
<td style="text-align:left;color: black !important;">
BEAST 1
</td>
<td style="text-align:left;color: black !important;">
N (n = 62)
</td>
<td style="text-align:left;color: black !important;">
534
</td>
<td style="text-align:left;color: black !important;">
Passive
</td>
<td style="text-align:left;color: black !important;">
Migration rates, Root state
</td>
<td style="text-align:left;color: black !important;">
<a href="https://doi.org/10.4142/jvs.2015.16.4.459">Dibia et al., 2015</a>
</td>
</tr>
<tr>
<td style="text-align:left;color: black !important;">
Bayesian discrete phylogeography
</td>
<td style="text-align:left;color: black !important;">
Symmetric DTA + BSSVS
</td>
<td style="text-align:left;color: black !important;">
BEAST 1
</td>
<td style="text-align:left;color: black !important;">
N (n = 203)
</td>
<td style="text-align:left;color: black !important;">
1353
</td>
<td style="text-align:left;color: black !important;">
GenBank
</td>
<td style="text-align:left;color: black !important;">
Migration rates
</td>
<td style="text-align:left;color: black !important;">
<a href="https://doi.org/10.1371/journal.pntd.0003502">Yao et al., 2015</a>
</td>
</tr>
<tr>
<td style="text-align:left;color: black !important;">
Bayesian phylodynamics
</td>
<td style="text-align:left;color: black !important;">
Coalescent models
</td>
<td style="text-align:left;color: black !important;">
BEAST 1
</td>
<td style="text-align:left;color: black !important;">
Whole genome (n = 248)
</td>
<td style="text-align:left;color: black !important;">
</td>
<td style="text-align:left;color: black !important;">
GenBank<br>WHO Collaborative Center for Reference and Research on Rabies<br>French National Reference Centre for Rabies
</td>
<td style="text-align:left;color: black !important;">
Nucleotide substitution rate, tMRCA
</td>
<td style="text-align:left;color: black !important;">
<a href="https://doi.org/10.1371/journal.ppat.1006041">Troupin et al., 2016</a>
</td>
</tr>
<tr>
<td style="text-align:left;color: black !important;">
Bayesian discrete phylogeography
</td>
<td style="text-align:left;color: black !important;">
Asymmetric DTA + BSSVS
</td>
<td style="text-align:left;color: black !important;">
BEAST 1
</td>
<td style="text-align:left;color: black !important;">
N (n = 401)
</td>
<td style="text-align:left;color: black !important;">
1350
</td>
<td style="text-align:left;color: black !important;">
Active<br>Passive
</td>
<td style="text-align:left;color: black !important;">
Nucleotide substitution rate, Markov jumps, Migration rates
</td>
<td style="text-align:left;color: black !important;">
<a href="https://doi.org/10.1186/s12985-017-0769-6">Zhang et al., 2017</a>
</td>
</tr>
<tr>
<td style="text-align:left;color: black !important;">
Bayesian discrete phylogeography
</td>
<td style="text-align:left;color: black !important;">
Asymmetric DTA + BSSVS
</td>
<td style="text-align:left;color: black !important;">
BEAST 1
</td>
<td style="text-align:left;color: black !important;">
N (n = 20)<br>G (n = 16)
</td>
<td style="text-align:left;color: black !important;">
1353<br>1575
</td>
<td style="text-align:left;color: black !important;">
Passive
</td>
<td style="text-align:left;color: black !important;">
Nucleotide substitution rate, Migration rates, tMRCA
</td>
<td style="text-align:left;color: black !important;">
<a href="https://doi.org/10.1002/jmv.24769">Ma et al., 2017</a>
</td>
</tr>
<tr>
<td style="text-align:left;color: black !important;">
Bayesian continuous phylogeography
</td>
<td style="text-align:left;color: black !important;">
Relaxed random walk (lognormal prior)
</td>
<td style="text-align:left;color: black !important;">
BEAST 1
</td>
<td style="text-align:left;color: black !important;">
N, P, intergenic G-L (n = 250)
</td>
<td style="text-align:left;color: black !important;">
3080
</td>
<td style="text-align:left;color: black !important;">
Talbi et al., 2009
</td>
<td style="text-align:left;color: black !important;">
Velocity, distance and duration of movement<br>Velocity-associated landscape features
</td>
<td style="text-align:left;color: black !important;">
<a href="https://doi.org/10.1093/molbev/msx176">Dellicour et al., 2017</a>
</td>
</tr>
<tr>
<td style="text-align:left;color: black !important;">
Bayesian discrete phylogeography<br>Bayesian continuous phylogeography
</td>
<td style="text-align:left;color: black !important;">
Asymmetric GLM + BSSVS<br>Relaxed random walk (gamma prior)
</td>
<td style="text-align:left;color: black !important;">
BEAST 1
</td>
<td style="text-align:left;color: black !important;">
Whole genome (n = 152)
</td>
<td style="text-align:left;color: black !important;">
</td>
<td style="text-align:left;color: black !important;">
Brunker et al., 2015<br>Not indicated
</td>
<td style="text-align:left;color: black !important;">
Nucleotide substitution rate, Markov jumps, Migration rates<br>Velocity, distance and duration of movement <br>Velocity-associated landscape features
</td>
<td style="text-align:left;color: black !important;">
<a href="https://doi.org/10.1111/mec.14470">Brunker et al., 2018</a>
</td>
</tr>
<tr>
<td style="text-align:left;color: black !important;">
Bayesian discrete phylogeography
</td>
<td style="text-align:left;color: black !important;">
Symmetric DTA + BSSVS
</td>
<td style="text-align:left;color: black !important;">
BEAST 1
</td>
<td style="text-align:left;color: black !important;">
N (n = 112)
</td>
<td style="text-align:left;color: black !important;">
1353
</td>
<td style="text-align:left;color: black !important;">
GenBank
</td>
<td style="text-align:left;color: black !important;">
Nucleotide substitution rate, Migration rates, Population dynamics, tMRCA
</td>
<td style="text-align:left;color: black !important;">
<a href="https://doi.org/10.1007/s00705-019-04297-8">Wang et al., 2019</a>
</td>
</tr>
<tr>
<td style="text-align:left;color: black !important;">
Bayesian discrete phylogeography<br>Bayesian continuous phylogeography
</td>
<td style="text-align:left;color: black !important;">
Symmetric DTA + BSSVS<br>Relaxed random walk (not indicated)
</td>
<td style="text-align:left;color: black !important;">
BEAST 1
</td>
<td style="text-align:left;color: black !important;">
Whole genome (n = 101)
</td>
<td style="text-align:left;color: black !important;">
</td>
<td style="text-align:left;color: black !important;">
Passive
</td>
<td style="text-align:left;color: black !important;">
Nucleotide substitution rate<br>Direction of spread<br>Velocity, distance and duration of movement<br>Velocity- and direction-associated landscape features
</td>
<td style="text-align:left;color: black !important;">
<a href="https://doi.org/10.1111/mec.15222">Dellicour et al., 2019</a>
</td>
</tr>
</tbody>
<tfoot>
<tr>
<td style="padding: 0; " colspan="100%">
<span style="font-style: italic;">Note: </span>
</td>
</tr>
<tr>
<td style="padding: 0; " colspan="100%">
<sup></sup> Abbreviations: ACCTRAN, Accelerated Transformation; BSSVS, Bayesian Stochastic Search Variable Selection; DELTRAN, Delayed Transformation; DTA, Discrete Trait Analysis; GLM, Generalized Linear Model; tMRCA, time to the Most Recent Common Ancestor; WHO World Health Organization
</td>
</tr>
<tr>
<td style="padding: 0; " colspan="100%">
<sup>a</sup> Symmetry not indicated in the original publication
</td>
</tr>
</tfoot>
</table>
</div>




</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.odd').parent('tbody').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- tabsets -->

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open');
  });
});
</script>

<!-- code folding -->


<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
